src/amber/cli/templates/error/controller.cr

```diff
@@ -0,0 +1,21 @@
+module Amber::CLI::ErrorTemplate
+  class Controller < Teeplate::FileTree
+    include Amber::CLI::Helpers
+    directory "#{__DIR__}/controller"
+
+    @name : String
+    @actions : Hash(String, String)
+    @language : String
+
+    def initialize(@name, actions)
+      @language = fetch_language
+      @actions = actions.map{|action| [action, "get"] }.to_h
+      add_plugs
+    end
+
+    private def add_plugs
+      add_plugs :web, "plug Amber::Pipe::Error.new"
```

Comment:
Is this idempotent? I assumed it was, but haven't doublechecked yet.

Reply:
yes, it should only add it once.

Reply:
Do we need to specify `plug` to `add_plugs`? 😕